Makefile

```diff
@@ -0,0 +1,55 @@
+# See repo README for more details on Makefile targets
+
+make install:
+# Install dependencies
+	python3 -m venv venv
+	. venv/bin/activate
+	@echo
+	@echo "\033[95mInstalling dependencies . . .\033[0m"
+	@echo
+	pip install -r requirements.txt
+	@echo
+	@echo "\033[95mInstalling c7n project . . .\033[0m"
+	@echo
+	pip install -e git+https://github.com/cloud-custodian/cloud-custodian.git#egg=c7n
```

Comment:
Nice use of editable install to pick up mugc.py without needing to copy files or run an explicit git clone. Feels like a good fit for this repo :+1: